.travis.yml

```diff
@@ -39,4 +40,5 @@ script:
   - cd docs/guides/admin && mkdocs build && cd -
   - cd docs/guides/developer && mkdocs build && cd -
   - cd docs/guides/user && mkdocs build && cd -
+  - cd docs $$ grunt && cd -
```

Comment:
I'm guessing this should be docs *&&* grunt?

Reply:
yes, you are right.